moveit_planners/ompl/ompl_interface/src/detail/state_validity_checker.cpp

```diff
@@ -137,7 +136,7 @@ bool ompl_interface::StateValidityChecker::isValidWithoutCache(const ompl::base:
   collision_detection::CollisionResult res;
   planning_context_->getPlanningScene()->checkCollision(
       verbose ? collision_request_simple_verbose_ : collision_request_simple_, res, *kstate);
-  return res.collision == false;
+  return !res.collision;
```

Comment:
This is clang-tidy. It likes to simplify boolean logic where it can. See https://clang.llvm.org/extra/clang-tidy/checks/readability-simplify-boolean-expr.html